fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/materialization/MvRewriteStrategy.java

```diff
@@ -91,21 +90,13 @@ private boolean isEnableRBOSingleTableRewrite(OptExpression queryPlan) {
             }
             // If view delta is enabled and there are multi-table mvs, return false.
             // if mv has multi table sources, we will process it in memo to support view delta join rewrite
-            if (sessionVariable.isEnableMaterializedViewViewDeltaRewrite() &&
+            if (!sessionVariable.isEnableRuleBasedMaterializedViewRewrite() &&
```

Comment:
Why changes this?

Reply:
isEnableMaterializedViewViewDeltaRewrite is used to control view delta join, not single table rule based mv rewrite.